endpoints/currency_rates.go

```diff
@@ -0,0 +1,74 @@
+package endpoints
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/prebid/prebid-server/currencies"
+)
+
+// currencyRatesInfo holds currency rates information.
+type currencyRatesInfo struct {
```

Comment:
Took me a while to figure out why you have two info structs. Finally realized because they are private, but in different packages.

Reply:
Yes indeed, I introduced it in order not to have the endpoint package tied to the currency package implementation :) 
It's also easier to test